discovery_engine/test/integration/utils/helpers.dart

```diff
@@ -15,7 +15,12 @@
 import 'dart:io';
 
 import 'package:xayn_discovery_engine/discovery_engine.dart'
-    show Manifest, createManifestReader, Configuration, FeedMarket;
+    show
+        Configuration,
+        DiscoveryEngine,
+        FeedMarket,
+        Manifest,
+        createManifestReader;
```

Comment:
it's weird that dart complains about unsorted `import`s, but not about unsorted `show`s.
```suggestion
        Configuration,
        createManifestReader,
        DiscoveryEngine,
        FeedMarket,
        Manifest;
```